### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by mwasilew2**
* fix missing cross-platform build destination

Signed-off-by: Michal Wasilewski <mwasilewski@gmx.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,16 +23,18 @@ docker run \
 
 ## Building and running
 
-    git clone https://github.com/prometheus-community/postgres_exporter.git
-    cd postgres_exporter
-    make build
-    ./postgres_exporter <flags>
+    $ mkdir -p $GOPATH/src/github.com/prometheus-community
+    $ cd $GOPATH/src/github.com/prometheus-community
```
Comment (SuperQ) [inline README.md:27]: This is not necessary anymore, and not recommended. The `GOPATH` style is [officially deprecated](https://blog.golang.org/modules2019) (Go 1.13).

File: README.md [hunk 2]
```diff
@@ -23,16 +23,18 @@ docker run \
...
+    $ cd postgres_exporter
+    $ make build
+    $ ./postgres_exporter <flags>
 
 To build the Docker image:
 
-    make promu
-    promu crossbuild -p linux/amd64 -p linux/armv7 -p linux/amd64 -p linux/ppc64le
-    make docker
+    $ make promu
```
Comment (SuperQ) [inline README.md:35]: I'm not sure I like adding command prompts to this. I find it easier to copy-n-paste lines without them.

**Comment by mwasilew2** [comment]
thanks @SuperQ , this is ready for another look, the only change is fixing a typo in build destination

**Comment by mwasilew2** [comment]
This was fixed in: https://github.com/prometheus-community/postgres_exporter/pull/903

**Pull request closed without merging**